regression/contracts/assigns_replace_havoc_dependent_targets/vect.h

```diff
@@ -0,0 +1,48 @@
+#include <assert.h>
+#include <stdlib.h>
+
+typedef struct vect
+{
+  char *arr;
+  size_t size;
+} vect;
+
+void resize_vec(vect *v, size_t incr)
+  // clang-format off
+__CPROVER_requires(
+  __CPROVER_is_fresh(v, sizeof(vect)) &&
+  0 < v->size && v->size <= __CPROVER_max_malloc_size &&
```

Comment:
Why do you need to check whether `v->size` is smaller than `__CPROVER_max_malloc_size`? Haven't this problem been solved (see https://github.com/diffblue/cbmc/issues/6130)?